Article/Article.js

```diff
@@ -1,29 +1,27 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.article.querySelector('.menu-button'); 
```

Comment:
here we want to reference the `.expandButton` class, not the `.menu-button` class